companies/fidelity-de.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
The legal notice (https://www.fidelity.de/impressum/) lists an email (info@fidelity.de) that we should include.